testbed/tests/window/test_window.py

```diff
@@ -144,11 +145,98 @@ async def test_move_and_resize(main_window, main_window_probe, capsys):
 
     async def test_full_screen(main_window, main_window_probe):
         """Window can be made full screen"""
-        main_window.full_screen = True
-        await main_window_probe.wait_for_window("Full screen is a no-op")
+        with pytest.warns(
+            DeprecationWarning,
+            match="`Window.full_screen` is deprecated. Use `Window.state` instead.",
+        ):
+            main_window.full_screen = True
+        await main_window_probe.wait_for_window("Main window is in full screen")
+
+        with pytest.warns(
+            DeprecationWarning,
+            match="`Window.full_screen` is deprecated. Use `Window.state` instead.",
+        ):
+            main_window.full_screen = False
+        await main_window_probe.wait_for_window(
+            "Main window is no longer in full screen"
+        )
+
+    async def test_window_state_minimized(main_window, main_window_probe):
+        """Window can have minimized window state"""
+        # WindowState.MINIMIZED is unimplemented on both Android and iOS.
```

Comment:
There's no need to annotate this. It's clear from the skip in the test run.

Reply:
Removed it.